react/features/blur/functions.js

```diff
@@ -0,0 +1,13 @@
+// @flow
+
+/**
+ * Returns true if video blur effect is enabled or false
+ * otherwise.
+ *
+ * @param {Object} state - Redux state
+ *
+ * @returns {boolean}
+ */
+export function isVideoBlurEnabled(state: Object) {
```

Comment:
Do you need this function? It is an one liner and you are using it on 1 place only...

Reply:
Removed functions.js